CONTRIBUTING.md

```diff
@@ -0,0 +1,140 @@
+
+# Contributing guidelines
+
+## Rendering notebooks
+
+This repository uses [snakemake](https://snakemake.readthedocs.io/en/stable/getting_started/installation.html) to render the all notebooks.
+The `Snakefile` calls the `scripts/render-notebooks.R` which renders the notebooks but leaves these `.Rmd` files still be individually downloaded by a user and [ran without the `pandoc` error](https://github.com/AlexsLemonade/refinebio-examples/pull/148#issuecomment-669170681).=
+`snakemake` should be ran after changes have been made and before any `Pull Request` are filed.
+
+### How to re-render the notebooks
+
+**Step 1)** Install snakemake (if you haven't before)
+Follow the installation instructions on the [snakemake docs](https://snakemake.readthedocs.io/en/stable/getting_started/installation.html).
+
+**Step 2)** Add any new `.Rmd` notebooks to `input` section of the `Snakefile` as a file paths relative to the `Snakefile`.
+
+**Step 3)** Run the thing!
+Navigate to the `refinebio-examples` repository.
+
+Activate snakemake
+```
+conda activate snakemake
+```
+Run it!
+```
+snakemake --cores 1
+```
+If ran successfully, it will spit out a log and all the html output files will have nicely rendered citations.
+
+### About the render-notebooks.R script
+
+The `render-notebooks.R` script adds a `bibliography:` specification in the `.Rmd` header so all citations are automatically rendered.
+
+**Options:**
+- `--rmd`: provided by snakemake, the input `.Rmd` file to render.   
+- `--bib_file`: File path for the  `bibliography:` header option.
+Default is the `references.bib` script at the top of the repository.  
+- `--html`: Default is to save the output `.nb.html` file of the same name as the input `.Rmd` file. This option allows you to specify an output file name. Default is used by snakemake.   
```

Comment:
Since we are using `html_document`, the extension should just be `.html` (no `.nb`)
```suggestion
- `--html`: Default is to save the output `.html` file of the same name as the input `.Rmd` file. This option allows you to specify an output file name. Default is used by snakemake.   
```

Reply:
> > This is actually not what you will want to do in the end... You will add the `.html` file names that _will be created_ to the
> > ```
> > rule target:
> >     input:
> > ```
> > 
> > 
> > section (comma separated, in quotes).
> 
> We can make this change over on #150 ?

Yes, but the instructions here will also need to change.

Reply:
Added this to the steps!